test/metabase/sync/field_values_test.clj

```diff
@@ -54,6 +54,27 @@
       (is (= [1 2 3 4]
              (venues-price-field-values))))))
 
+(deftest sync-should-properly-handle-last-used-at
+  (testing "Test that syncing will skip updating inactive FieldValues"
+    (db/update! FieldValues
+                (db/select-one-id FieldValues :field_id (mt/id :venues :price) :type :full)
+                :last_used_at (t/minus (t/offset-date-time) (t/days 20)))
+    (db/update! FieldValues (db/select-one-id FieldValues :field_id (mt/id :venues :price) :type :full) :values [1 2 3])
```

Comment:
is the purpose of the 2nd update is to update the values?

if it is, could we do both of them in one update?